### PR TITLE
feat(cdn): purge and deduplication

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

File: engine/cdn/storage/storageunit.go [hunk 1]
```diff
@@ -226,6 +226,8 @@ func (r *RunningStorageUnits) Start(ctx context.Context, gorts *sdk.GoRoutines)
 	// 	Feed the sync processes with a ticker
 	gorts.Run(ctx, "RunningStorageUnits.Start", func(ctx context.Context) {
 		tickr := time.NewTicker(time.Second)
+		tickrPurge := time.NewTicker(time.Second)
```
Comment (sguiheux) [inline engine/cdn/storage/storageunit.go:229]: missing defer Stop

**Review by sguiheux — CHANGES_REQUESTED** [review]

File: engine/cdn/storage/storageunit.go (hunk 1 above)
Comment (fsamin) [reply engine/cdn/storage/storageunit.go:229]: done

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14775.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: engine/api/api_routes.go [hunk 2]
```diff
@@ -45,6 +45,7 @@ func (api *API) InitRouter() {
 	r.Handle("/auth/consumer/{consumerType}/signin", Scope(sdk.AuthConsumerScopeAccessToken), r.POST(api.postAuthSigninHandler, service.OverrideAuth(api.authOptionalMiddleware), MaintenanceAware()))
 	r.Handle("/auth/consumer/{consumerType}/detach", Scope(sdk.AuthConsumerScopeAccessToken), r.POST(api.postAuthDetachHandler))
 	r.Handle("/auth/consumer/signout", ScopeNone(), r.POST(api.postAuthSignoutHandler))
+	r.Handle("/auth/session/{sessionID}", ScopeNone(), r.GET(api.getAuthSession))
```
Comment (richardlt) [inline engine/api/api_routes.go:48]: Only admin should be able to call this route.

**Review by richardlt — CHANGES_REQUESTED** [review]

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14795.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#14797.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Docs -   
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Pull request merged**